scripts/meta_writer.py

```diff
@@ -80,9 +80,9 @@
     "sub_pool_resources",
     "sub_band",
     "sub_product",
-    "m???_observer",
-    "m???_activity",
-    "m???_target",
+    "*_observer",
+    "*_activity",
+    "*_target",
```

Comment:
This is going to duplicate some keys e.g. cbf_target and {cn}_obs_activity. I'm not sure what the effect will be i.e. whether they'll end up twice in the .rdb file, and if that'll lead to any logging about setting keys twice. I think it might be better to pass in a list of antennas and instantiate a pattern for each antenna since this has the potential to pull in other random keys that we might not want.

If you just want a quick short-term fix, maybe add 'S????_observer' etc to the list?

Reply:
Fair enough - I did test it and the duplicates didn't create an issue, but being more explicit is probably the path of least resistance. Have updated to use this approach.